### PR TITLE
Get rid of annotations

### PR DESCRIPTION
**Pull request by nikita-gorodeckij**

### CODE REVIEW

File: proco/locations/serializers.py [hunk 1]
```diff
@@ -51,10 +51,10 @@ def get_integration_status(self, instance):
         return instance.latest_status[0].integration_status if instance.latest_status else None
 
     def get_date_of_join(self, instance):
-        return getattr(instance, 'date_of_join', None)
+        return instance.first_status[0].created if instance.first_status else None
```
Comment (roman-karpovich) [inline proco/locations/serializers.py:54]: let's save date of join directly to the country so we won't need additional db query just to get one field. this information will be filled at once on first csv upload and then will never be changed

File: proco/connection_statistics/models.py [hunk 2]
```diff
@@ -45,6 +46,9 @@ class CountryWeeklyStatus(TimeStampedModel, models.Model):
     connectivity_speed = models.FloatField(blank=True, default=0.0)
     integration_status = models.PositiveSmallIntegerField(choices=INTEGRATION_STATUS_TYPES, default=JOINED)
     avg_distance_school = models.FloatField(blank=True, default=0.0)
+    schools_with_data_percentage = models.DecimalField(
```
Comment (roman-karpovich) [inline proco/connection_statistics/models.py:49]: there is no code to keep this field updated

File: proco/connection_statistics/models.py [hunk 3]
```diff
@@ -45,6 +46,9 @@ class CountryWeeklyStatus(TimeStampedModel, models.Model):
...
+        decimal_places=2, max_digits=6, default=0, validators=[MaxValueValidator(100), MinValueValidator(0)],
```
Comment (roman-karpovich) [inline proco/connection_statistics/models.py:50]: do we have reasons to validate automatically updated readonly fields?

**Review by roman-karpovich — CHANGES_REQUESTED** [review]

File: proco/connection_statistics/models.py (hunk 3 above)
Comment (nikita-gorodeckij) [reply proco/connection_statistics/models.py:50]: Is it bad to have additional validation?

Comment (roman-karpovich) [reply proco/connection_statistics/models.py:50]: nope. just interesting why now :)

**Pull request merged**